test/jasmine/tests/click_test.js

```diff
@@ -180,7 +180,9 @@ describe('Test click interactions:', function() {
 
     describe('drag interactions', function() {
         beforeEach(function(done) {
-            Plotly.plot(gd, mockCopy.data, mockCopy.layout).then(done);
+            Plotly.plot(gd, mockCopy.data, mockCopy.layout).then(function() {
+                setTimeout(done, 200);
```

Comment:
I couldn't adapt the solution to Firefox, because I couldn't find a flag similar to Chrome's `--window-size`.

Using @monfera 's trick I got to enable webgl in my VM's chromium, like this:
```
user@host:~/github/plotly.js$ git diff
diff --git a/test/jasmine/karma.conf.js b/test/jasmine/karma.conf.js
index 5905271..52e1ce4 100644
--- a/test/jasmine/karma.conf.js
+++ b/test/jasmine/karma.conf.js
@@ -69,7 +69,15 @@ func.defaultConfig = {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['Chrome'],
+    browsers: ['Chrome_WindowSized'],
+
+    // custom browser options
+    customLaunchers: {
+        Chrome_WindowSized: {
+            base: 'Chrome',
+            flags: ['--window-size=1035,617', '--enable-webgl', '--ignore-gpu-blacklist'] // values came from observing default size; all test cases pass with it
+        }
+    },
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
```

I ran the tests using Chromium and the drag-interaction tests work, but one of the gl tests failed (the one that used to fail with Firefox). See:
```
user@host:~/github/plotly.js$ npm run test-jasmine

> plotly.js@1.9.0 test-jasmine /home/user/github/plotly.js
> karma start test/jasmine/karma.conf.js

20 04 2016 16:17:53.134:INFO [framework.browserify]: registering rebuild (autoWatch=true)
20 04 2016 16:18:02.923:INFO [framework.browserify]: 9104144 bytes written (8.99 seconds)
20 04 2016 16:18:02.988:INFO [framework.browserify]: bundle built
20 04 2016 16:18:03.009:WARN [karma]: No captured browser, open http://localhost:9876/
20 04 2016 16:18:03.028:INFO [karma]: Karma v0.13.22 server started at http://localhost:9876/
20 04 2016 16:18:03.034:INFO [launcher]: Starting browser Chrome
20 04 2016 16:18:05.306:INFO [Chromium 49.0.2623 (Ubuntu 0.0.0)]: Connected on socket /#f3LcZeKt1FWNckeOAAAA with id 18105360
Chromium 49.0.2623 (Ubuntu 0.0.0) Test gl plot interactions gl3d plots scatter3d click events should have FAILED
	TypeError: Cannot convert undefined or null to object
	    at Object.<anonymous> (/tmp/b519d49625b3b7b09b7d4b9b755aee99.browserify:104577:31 <- tests/gl_plot_interact_test.js:141:0)
LOG: 'lost context'
LOG: 'lost context'
LOG: 'lost context'
LOG: 'lost context'
LOG: 'restyle fail', undefined, '', undefined
LOG: 'relayout fail', undefined, false
LOG: 'type newtype already registered'
LOG: 'Oops, tried to put a polar trace on an incompatible graph of cartesian data. Ignoring this dataset.', Object{type: 'newtype', r: 'this is polar'}
Chromium 49.0.2623 (Ubuntu 0.0.0): Executed 773 of 773 (1 FAILED) (1 min 8.793 secs / 1 min 8.043 secs)
```

Shall I drop 04ef33edc6bb15d12fbdc7915fac767447e9a1b6 from this PR?